library/core/class.form.php

```diff
@@ -2627,7 +2628,7 @@ protected function _attributesToString($Attributes) {
             foreach ($Attributes as $Attribute => $Value) {
                 // Ignore reserved attributes
                 if (!in_array(strtolower($Attribute), $ReservedAttributes)) {
-                    $Return .= ' '.$Attribute.'="'.htmlspecialchars($Value, ENT_COMPAT, 'UTF-8').'"';
+                    $Return .= ' '.$Attribute.($Value===true ? '' : '="'.htmlspecialchars($Value, ENT_COMPAT, 'UTF-8').'"');
```

Comment:
Needs a space on either side of the `===` .
http://docs.vanillaforums.com/developers/contributing/coding-standard/